asf/sam0/include/samd21/samd21e15a.h

```diff
@@ -62,15 +62,15 @@ typedef volatile       uint32_t RwReg;   /**< Read-Write 32-bit register (volati
 typedef volatile       uint16_t RwReg16; /**< Read-Write 16-bit register (volatile unsigned int) */
 typedef volatile       uint8_t  RwReg8;  /**< Read-Write  8-bit register (volatile unsigned int) */
 #if !defined(_UL)
```

Comment:
This should be changed to check for `__UL` as well.

Reply:
updated in all header files that were changed